azure-spring-boot-parent/pom.xml

```diff
@@ -88,6 +98,27 @@
                 <artifactId>commons-logging</artifactId>
                 <version>${commons.logging.version}</version>
             </dependency>
+            <dependency>
+                <groupId>io.micrometer</groupId>
+                <artifactId>micrometer-registry-azure-monitor</artifactId>
+                <version>${azuremonitor.micrometer.registry}</version>
+            </dependency>
+            <dependency>
+                <groupId>io.micrometer</groupId>
+                <artifactId>micrometer-core</artifactId>
+                <version>${micrometer.core}</version>
+            </dependency>
+            <dependency>
+                <groupId>org.springframework.boot</groupId>
+                <artifactId>spring-boot-actuator-autoconfigure</artifactId>
+                <version>2.0.3.RELEASE</version>
+                <exclusions>
```

Comment:
As I mentioned in PR description even after explicitly excluding Micrometer-core transitive dependency from modules and adding the required version explicitly the build doesn't produce the desired output and when the bits are used in another test app, I still get 1.0.5 version imported.